cypress/integration/admin/fileManager/uploadImage.spec.js

```diff
@@ -20,18 +68,288 @@ context("File Manager View - CRUD", () => {
+        cy.findByTestId("fm-image-editor-dialog")
+            .should("be.visible")
+            .within(() => {
+                cy.get('[data-testid="flip-item"]').click();
+                cy.get("span").contains("Apply").click();
+                cy.findByText("Save").click();
+            });
+        cy.contains("File upload complete.").should("be.visible");
+
+        //  Assert that there are displayed both initial item and edited one.
+        cy.get('div[data-testid="fm-list-wrapper-file"]:nth-child(1)')
+            .find("div.label")
+            .contains(fileName);
+        cy.get('div[data-testid="fm-list-wrapper-file"]:nth-child(2)')
+            .find("div.label")
+            .contains(fileName);
+    });
+
+    it("should test minimum and maximum allowed file upload size", () => {
+        cy.visit("/settings/file-manager/general");
+
+        const fileDetails = [
+            {
+                maximumFileUploadBytesLimit: 500,
+                expectedUploadMessage:
+                    "sample.jpeg: Your proposed upload exceeds the maximum allowed size"
+            },
+            {
+                maximumFileUploadBytesLimit: 26214401,
+                expectedUploadMessage: "File upload complete."
+            }
+        ];
+
+        fileDetails.forEach(fileDetail => {
+            cy.visit("/settings/file-manager/general");
+            cy.findByLabelText("Maximum file upload size")
+                .clear()
+                .type(fileDetail.maximumFileUploadBytesLimit);
+            cy.contains("Save Settings").click();
+            cy.contains("Settings updated successfully.").should("be.visible");
+
+            cy.findByTestId("apps-menu").click();
+            cy.contains("Settings updated successfully.").should("not.exist");
+            cy.findByTestId("admin-drawer-footer-menu-file-manager").click();
+            cy.findByTestId("fm-list-wrapper").dropFile("sample.jpeg", "image/jpeg");
+
+            cy.contains(fileDetail.expectedUploadMessage).should("exist");
+        });
+    });
+
+    it("should test adding duplicate tag", () => {
+        const fileName = "sample_2.jpeg";
+
+        // Check if there are existing file and delete them.
+        cy.fmListFiles({}).then(files => {
+            for (let i = 0; i < files.length; i++) {
+                deleteFile();
+            }
+        });
+
+        // Drop file
+        cy.findByTestId("fm-list-wrapper").dropFile(fileName, "image/jpeg");
+        cy.findByText("File upload complete.").should("exist");
+
+        // Open file details
+        cy.findByTestId("fm-list-wrapper").within(() => {
+            cy.findAllByTestId("fm-list-wrapper-file")
+                .first()
+                .within(() => {
+                    cy.findByTestId("fm-file-wrapper-file-info-icon").click({ force: true });
+                });
+        });
+
+        cy.get("span").contains("File details").should("be.visible");
```

Comment:
Can we use `findByText` here?

P.S. Not sure if you've seen it, but we're also using https://testing-library.com/docs/cypress-testing-library/intro/ that gives us additional selectors. This is where the mentioned `findByText` is coming from.